Documentation/ApiOverview/Database/QueryBuilder/Index.rst

```diff
@@ -853,7 +853,12 @@ Remarks:
     out the first n records". Internally, :sql:`LIMIT` will be added by
     Doctrine DBAL and set to a very high value.
 
-*   Use :php:`->setMaxResults(null)` to retrieve all results.
+*   In TYPO3 v13 it will be possible to use :php:`->setMaxResults(null)` to retrieve 
```

Comment:
I would prefer to document the "current" (12.4) state first, and only then mention v13. Maybe also state that if you want an unlimited query, you can leave out the setting altogether, or am I missing something?